desktop.css

```diff
@@ -396,4 +398,76 @@
     width: 48px;
     height: 48px;
   }
+
+  /* //-----------popup-------------------------- */
+  .popup-displaycard {
+    /* width: 100%;
+    height: 100%; */
+    flex-direction: column;
+    left: 1px;
+    overflow-y: scroll;
+    padding-right: 64px;
+  }
+
+  .workspop {
+    display: flex;
+    gap: 25px;
+    justify-content: space-around;
+  }
+
+  .position-descr-dv {
+    gap: 22px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .canopypop {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #7a869a;
+  }
+
+  .imgPopupdesc {
+    display: flex;
+    align-content: center;
+    width: 80%;
+    height: 90%;
```

Comment:
- [Optional] Good gob for giving fixed size for your modal image, I think 80% and 90% is high almost your entire screen, I think it is better to minimize it around 50%. 👌👌